### PR TITLE
Upgrade to FoamTree 3.5.0, now available as an NPM package.

### PR DESCRIPTION
**Pull request by stanislawosinski**
The latest FoamTree releases fix various functional bugs and performance regressions in the flattened display mode used by webpack-bundle-analyzer.

### CODE REVIEW

**Comment by stanislawosinski** [comment]
I've signed the CLA some time ago, but the check still shows "Pending". Is there anything else I need to do in terms of the CLA?

**Comment by valscion** [comment]
Sometimes the CLA check gets stuck ☺️. Luckily we can close+reopen and then the CLA check gets retried — now it's passing.

File: webpack.config.js [hunk 1]
```diff
@@ -23,7 +23,6 @@ module.exports = opts => {
 
     resolve: {
       modules: [
-        `${__dirname}/client/vendor`,
```
Comment (th0r) [inline webpack.config.js:26]: Let's remove `resolve.modules` completely as `node_modules` are used by default.

File: webpack.config.js [hunk 2]
```diff
@@ -60,7 +59,7 @@ module.exports = opts => {
       rules: [
         {
           test: /\.jsx?$/u,
-          exclude: /(node_modules|client\/vendor)/u,
+          exclude: /(node_modules)/u,
```
Comment (th0r) [inline webpack.config.js:62]:
```suggestion
          exclude: /node_modules/u,
```

**Review by th0r — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 1 above)
Comment (stanislawosinski) [reply webpack.config.js:26]: Good point, I've just pushed the update.

**Comment by stanislawosinski** [comment]
I'll resolve CHANGELOG.md conflicts when you merge #414.

**Comment by stanislawosinski** [comment]
Is there anything else for me to correct for this PR to get merged?

**Comment by th0r** [comment]
> Is there anything else for me to correct for this PR to get merged?

No, thanks!

**Pull request merged**